PetProject/Screens/SearchStackScreens/TopDealsList/StoreSortedDealsListCell.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  StoreSortedDealsListCell.swift
+//  PetProject
+//
+//  Created by Тимофей Лукашевич on 4.05.22.
+//
+
+import SwiftUI
+
+struct StoreSortedDealsListCell: View {
+    
+    @ObservedObject var viewModel: StoreSortedListCellViewModel
```

Comment:
Here we have to use @ObservedObject or @StateObject? And why?

Reply:
Done